src/pybind/mgr/dashboard/controllers/cluster_configuration.py

```diff
@@ -55,3 +55,8 @@ def create(self, name, value):
                     break
             else:
                 CephService.send_command('mon', 'config rm', who=section, name=name)
+
+    def bulk_set(self, options):
+        for name, value in options.items():
+            CephService.send_command('mon', 'config set', who=value['section'],
```

Comment:
There is already `mgr_module.MgrModule#set_config`. Does it make sense to just use that instead?

Reply:
Are you sure that ``set_config`` is similar to ``config set``? E.g. ``set_config`` takes two arguments ``key`` and ``val`` (both strings). ``config set`` takes ``section``, ``name`` and ``value``.

Reply:
Haven't used it till now, but `set config` sounds bloody similar to `config set`

Reply:
It's been a while, but I've used `mgr_module.MgrModule#set_config` for the settings implementation in `dashboard/settings.py`. It seems to me as if this is the library version of what mgr modules can use to manage their own settings. The `send_command` approach in contrast is the one which can be used to manage all settings, including those of other mgr modules.

Reply:
I'm going to resolve the conversation here. If that's not correct feel free to re-open it.